### PR TITLE
Add feature flag and `LinkConsumerIncentive` struct

### PR DESCRIPTION
**Pull request by tillh-stripe**
## Summary

This pull request adds a feature flag for Instant Bank Payments Incentives, and adds the `LinkConsumerIncentive` struct in `ElementsSession`.

## Motivation

[CONSUMERBANK-567](https://jira.corp.stripe.com/browse/CONSUMERBANK-567)

### CODE REVIEW

File: StripePayments/StripePayments/Source/API Bindings/Models/Shared/LinkConsumerIncentive.swift [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+//
+
+import Foundation
+
+@_spi(STP) public final class LinkConsumerIncentive: NSObject, STPAPIResponseDecodable {
+    
+    @_spi(STP) public let incentiveParams: IncentiveParams
+    @_spi(STP) public let incentiveDisplayText: String?
+    
+    @_spi(STP) public let allResponseFields: [AnyHashable : Any]
```
Comment (mats-stripe) [inline StripePayments/StripePayments/Source/API Bindings/Models/Shared/LinkConsumerIncentive.swift:15]: Just curious, why do we want to hold on to all response fields here?

**Review by mats-stripe — APPROVED** [review]
Nice, LGTM!

File: StripePayments/StripePayments/Source/API Bindings/Models/Shared/LinkConsumerIncentive.swift (hunk 1 above)
Comment (tillh-stripe) [reply StripePayments/StripePayments/Source/API Bindings/Models/Shared/LinkConsumerIncentive.swift:15]: It’s part of `STPAPIResponseDecodable`, although I’m not so sure of its usefulness here.

Comment (tillh-stripe) [reply StripePayments/StripePayments/Source/API Bindings/Models/Shared/LinkConsumerIncentive.swift]: Addressed in e6fee128476533b806d57b38eac4128f2e746fde.

**Pull request merged**